views/topic/topic.hbs

```diff
@@ -167,21 +162,16 @@
                         {{#userHasPermission "HOMEWORK_CREATE"}}
                             <a href="/homework/new?course={{../courseId}}&topic={{../_id}}&private=true"
                                 class="btn btn-secondary btn-sm pull-right ml-1" data-toggle="tooltip"
-                                title="{{$t "global.button.addPrivateTask"}}">
-                                <i class="fa fa-plus" aria-hidden="true"></i><span class="hidden-md-down"> {{$t "global.button.addPrivateTask" }}</span>
+                                title="{{$t "global.button.addDraftTask"}}">
+                                <i class="fa fa-plus" aria-hidden="true"></i><span class="hidden-md-down"> {{$t "global.button.addDraftTask" }}</span>
```

Comment:
In `topics.hbs` in lines 224 & 225 translation key `"global.button.addPrivateTask"` still needs to be changed

Reply:
I changed it.